runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -8964,11 +8979,13 @@ makeMixedMemSetGraph(TR::Compilation *c, int32_t ctrl)
    return tgt;
    }
 
+//////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////
+
 
 
 
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
```

Comment:
Please add back these two lines with all slashes that were accidentally removed:

```
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
```